tests/e2e/actor_features/actor_features_test.go

```diff
@@ -368,23 +370,30 @@ func TestActorFeatures(t *testing.T) {
 		// Min call is based off of having a 1s period/due time, the amount of seconds we've waited, and a bit of room for timing.
 		require.GreaterOrEqual(t, firstCount, minFirstCount)
 
-		err = tr.Platform.Restart(appName)
-		assert.NoError(t, err)
-		externalURL = tr.Platform.AcquireAppExternalURL(appName)
-		require.NotEmpty(t, externalURL, "Could not get external URL after app restart.")
+		t.Logf("Restarting %s ...", appName)
+		err := tr.Platform.Restart(appName)
+		require.NoError(t, err)
 
-		time.Sleep(sleepTime)
+		time.Sleep(120 * time.Second)
```

Comment:
Is this really necessary? Or was this something you were trying while debugging?

Reply:
I don't think this is necessary anymore.